Sources/TuistGenerator/Generator/SchemesGenerator.swift

```diff
@@ -161,6 +161,18 @@ final class SchemesGenerator: SchemesGenerating {
         var testables: [XCScheme.TestableReference] = []
         var preActions: [XCScheme.ExecutionAction] = []
         var postActions: [XCScheme.ExecutionAction] = []
+        var codeCoverageTargets: [XCScheme.BuildableReference] = []
+        
+        testAction.codeCoverageTargets.forEach { name in
```

Comment:
What about extracting this into a method:

```swift
func testCoverageTargetReferences(...) -> [XCScheme.BuildableReference] {

}
```

Reply:
Done!